.github/workflows/jira-pr-title.yml

```diff
@@ -2,36 +2,37 @@ name: "Jira PR Title Check"
 on:
   pull_request:
     types: [opened, reopened, synchronize]
+  workflow_dispatch:
 jobs:
   check-title:
     runs-on: arc-runner-set
     steps:
-    - name: Check PR title
-      uses: actions/github-script@v7
-      with:
-        github-token: ${{ secrets.GITHUB_TOKEN }}
-        retries: 3
-        script: |
-          let data;
+      - name: Check PR title
```

Comment:
my vscode indented the code on save. This indentation looks closer to GitHub Docs. So maybe it's better?
![image](https://github.com/user-attachments/assets/a7ae7d84-3d90-4401-bba7-b5fe0dca2c9a)